### PR TITLE
Always restart etcd after an update request has completed.

### PR DESCRIPTION
**Pull request by masa213f**
Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

**Comment by ymmt2005** [comment]
```
Neco join/remove 
  should remove boot-3
  /root/go/src/github.com/cybozu-go/neco/dctest/join_remove_test.go:147
START: 2021-03-11T22:41:47Z
STEP: Running neco leave 3
STEP: Waiting for the request to complete
STEP: Confirming boot-3 gets removed from etcd
END: 2021-03-11T22:45:00Z

• Failure [192.488 seconds]
Neco
/root/go/src/github.com/cybozu-go/neco/dctest/suites_test.go:35
  join/remove
  /root/go/src/github.com/cybozu-go/neco/dctest/suites_test.go:80
    should remove boot-3 [It]
    /root/go/src/github.com/cybozu-go/neco/dctest/join_remove_test.go:147

    Expected
        <string>: boot-3
    not to equal
        <string>: boot-3

    /root/go/src/github.com/cybozu-go/neco/dctest/join_remove_test.go:167
```
https://app.circleci.com/pipelines/github/cybozu-go/neco/4276/workflows/9deff0e6-969c-47a5-9f35-f46463591d22/jobs/13071

File: dctest/join_remove_test.go [hunk 1]
```diff
@@ -149,18 +149,12 @@ func testJoinRemove() {
...
+		waitRequestComplete("members: [0 1 2]")
+
 		By("Waiting boot-3 gets removed from etcd")
 		Eventually(func() error {
-			// Sometimes, neco-worker aborts the update process.  We need to detect it and recover.
-			stdout, _, err := execAt(bootServers[0], "neco", "status")
-			if err != nil {
-				return err
-			}
-			if bytes.Contains(stdout, []byte("status: aborted")) {
```
Comment (ymmt2005) [inline dctest/join_remove_test.go:159]: Shouldn't we keep `neco recover` to avoid instabilities?

File: dctest/run_test.go [hunk 2]
```diff
@@ -182,6 +182,13 @@ func waitRequestComplete(check string) {
 			return fmt.Errorf("stdout: %s, stderr: %s, err: %v", stdout, stderr, err)
 		}
 		out := string(stdout)
+
+		// Sometimes, neco-worker aborts the update process.  We need to detect it and recover.
```
Comment (ymmt2005) [inline dctest/run_test.go:186]: Moving this here would hide other bugs than etcd instabilities...

Comment (masa213f) [reply dctest/run_test.go]: I fixed it in a51e09d.
Only if the `recover` argument is set, `neco recover` will be executed.

(Sorry I force pushed b881bae due to the commit message was wrong. But the contents is not changed.)

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**